dataprofiler/data_readers/csv_data.py

```diff
@@ -138,6 +137,20 @@ def _load_data_from_file(self, input_file_path):
             if not self._header:
                 self._header = self._determine_has_header(data_as_str)
                 self._checked_header = True
+
+        # if the delimiter is always at the end of each row,
+        # without any subsequent pattern, set delimiter to None
+        if self._delimiter:
```

Comment:
Since on line 135 we set self._delimiter if it is None, it will always be not None by this line, so this if statement will always be executed, couldn't we then remove the if statement?

Reply:
the above delimiter detection may return None, so, I have this if statement

Reply:
Oh okay, that makes sense, disregard this comment then